### ISSUE TITLE
[Vertical Navigation] WCAG2AA - verticalnavigation embed input component errors

### ISSUE THREAD

**Issue by JoaoEnesGoncalves** [lumada-design/hv-uikit-react#780]
**Describe the bug**
verticalnavigation embedded input component errors on inputText label and inputText name

**To Reproduce**
sample component:
- https://pentaho.github.io/hv-uikit-react/?path=/story/components-navigation-system--vertical-navigation
![vn](https://user-images.githubusercontent.com/9842927/68405168-88700480-0177-11ea-99a2-11d24350831a.PNG)

1.  run
```
node C:\Git\hv-uikit-react\automation\robot\storybook\_resources/accessibility.js --url=http://localhost:9002/iframe.html?id=coreverticalnavigation--verticalnavigation3 --standard=WCAG2AA --element="#root>div.Component-content-4" --actions="wait for element div[class|='VerticalNavigation-listContainer'] to be visible"
```

**Screenshots**
```
{
  documentTitle: 'Storybook',
  pageUrl: 'http://localhost:9002/iframe.html?id=coreverticalnavigation--verticalnavigation3',
  issues: [{
      code: 'aria-input-field-name',
      type: 'error',
      typeCode: 1,
      message: 'ARIA input fields have an accessible name (https://dequeuniversity.com/rules/axe/3.4/aria-input-field-name?application=axeAPI)',
      context: '<ul id="" class="List-root-214" role="listbox"><li role="option" aria-selected...</ul>',
      selector: '#root > div:nth-child(2) > div > div > div > div > div > div > div:nth-child(2) > ul',
      runner: 'axe',
      runnerExtras: [Object]
    },
    {
      code: 'label',
      type: 'error',
      typeCode: 1,
      message: 'Form elements must have labels (https://dequeuniversity.com/rules/axe/3.4/label?application=axeAPI)',
      context: '<input class="MuiInputBase-input-202 MuiInput-input-187 HvInput-input-163 MuiInputBase-inputAdornedEnd-208" placeholder="Search" type="text" value="">',
      selector: '#root > div:nth-child(2) > div > div > div > div > div > div > div:nth-child(1) > div > div:nth-child(2) > input',
      runner: 'axe',
      runnerExtras: [Object]
    },
    {
      code: 'WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name',
      type: 'error',
      typeCode: 1,
      message: 'This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.',
      context: '<input class="MuiInputBase-input-202 MuiInput-input-187 HvInput-input-163 MuiInputBase-inputAdornedEnd-208" placeholder="Search" type="text" value="">',
      selector: '#root > div:nth-child(2) > div > div > div > div > div > div > div:nth-child(1) > div > div:nth-child(2) > input',
      runner: 'htmlcs',
      runnerExtras: {}
    },
    {
      code: 'WCAG2AA.Principle1.Guideline1_3.1_3_1.F68',
      type: 'error',
      typeCode: 1,
      message: 'This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.',
      context: '<input class="MuiInputBase-input-202 MuiInput-input-187 HvInput-input-163 MuiInputBase-inputAdornedEnd-208" placeholder="Search" type="text" value="">',
      selector: '#root > div:nth-child(2) > div > div > div > div > div > div > div:nth-child(1) > div > div:nth-child(2) > input',
      runner: 'htmlcs',
      runnerExtras: {}
    }
  ]
}

```
